lib/pf/ConfigStore.pm

```diff
@@ -165,6 +165,14 @@ format the id
 
 sub _formatId { return $_[1]; }
```

Comment:
Maybe this should be renamed to _formatSectionName or _cleanupSectionName to avoid confusion with _cleanupId